js/modules/loki_snode_api.js

```diff
@@ -92,7 +92,7 @@ class LokiSnodeAPI {
       );
       // Filter 0.0.0.0 nodes which haven't submitted uptime proofs
       const snodes = result.result.service_node_states.filter(
-        snode => snode.address !== '0.0.0.0'
+        snode => snode.public_ip !== '0.0.0.0'
       );
       this.randomSnodePool = snodes.map(snode => ({
         address: snode.public_ip,
```

Comment:
```suggestion
        ip: snode.public_ip,
```

Reply:
Ya good idea, updated

Reply:
(We'll have to remember somehow that this will be a .snode address when we revert to using lokinet)